### ISSUE TITLE
On applying reflow resolution "Node.js" button is not accessible with keyboard "TAB" or "Arrow" keys.: A11y_Playwright OSS releases_Reflow

### ISSUE THREAD

**Issue by kupatkar99** [microsoft/playwright.dev#1631]
"[Check out Accessibility Insights](https://nam06.safelinks.protection.outlook.com/?url=https://accessibilityinsights.io/&data=02%7c01%7cv-stfe%40microsoft.com%7cb67b2c4b646d4f9561a208d6f4b5c39b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c636965458850501301&sdata=mxhokIKNMb22llsjXHLgU3XZibj1Qfx37rpY4PU2sfE%3D&reserved=0)! - Identify accessibility bugs before check-in and make bug fixing faster and easier."

### GitHub Tags:
#A11yAuto;#A11yMAS;#A11yTCS;#Playwright OSS Releases;#BM_PlaywrightOSSReleases_Web_Dec2024;#A11ySev2;#ChromiumEdge;#DesktopWeb;#FTP;#Win11;#WCAG1.4.10;#Reflow;#A11yWCAG2.1;

### Environmental Details:
App Name: Playwright OSS releases
#URL: https://playwright.dev/

### OS & Browser Details:
Microsoft Windows 11 Enterprise
Version 26100.2605
Microsoft Edge
Version 131.0.2903.99 (Official build) (64-bit)

### Pre- Requisites to apply reflow: 

1. Open the web page / screen of the web application that is to be tested.
2. Open Chrome/Edge dev Inspect tools. Keyboard shortcut "Ctrl+Shift+I" 
3. Ensure devtools are customized to Dock location: "undock into separate window". Minimize devtools so they are not obscuring the page.
4. Zoom the browser window of the page to be tested to 200%.
5. Adjust the browser window's height and width to the required viewport of 320px*256px. As long as devtools remain open while resizing the viewport size will be displayed in the upper right corner of the browser so the size can be verified. 

### Alternatively:

1. Change the OS display settings to a resolution of 1280x1024 at 100% dpi scaling.
2. Open the web page/ screen of the web application that is to be tested.
3. Zoom browser 400%. 
4. Set the browser to **full screen**. Keyboard shortcut "F11" key.
5. If any browser sidebars are still visible (e.g. the Edge discover sidebar), these _must_ be closed.

### Repro Steps:
1.Hit the Url: https://playwright.dev/
2. Playwright page will get open.
3. Observe that on applying reflow resolution "Node.js" button is accessible with keyboard or not.

### Actual:
On applying reflow resolution "Node.js" button is not accessible with keyboard "TAB" or "Arrow" keys.

### Expected Result:
On applying reflow resolution "Node.js" button should be accessible with keyboard "TAB" or "Arrow" keys.

### User Impact:
Users who zooms the page for there better visibility will face difficulty and not be able to access the control if it is not accessible with keyboard.

### Attachments:
https://github.com/user-attachments/assets/d19c500c-7189-4145-a68a-cfd8116dfeb9

**Comment by mxschmitt** [comment]
Update: Created https://github.com/facebook/docusaurus/pull/10898 upstream with a fix. Takes some time until this fix lands on our production environment.

**Comment by mxschmitt** [comment]
Update: Depends on a new release of https://github.com/facebook/docusaurus.

**Comment by kapilvaishna** [comment]
#External; #ResolvedByTT;

**Comment by mxschmitt** [comment]
Fix should be deployed, please recheck.

**Comment by kupatkar99** [comment]
Verified the issue issue is fixed. Please find added attachment for reference.

https://github.com/user-attachments/assets/4f5f67fb-a77d-4130-bf53-c6507ce72c98
